src/devtools/client/inspector/markup/actions/eventTooltip.ts

```diff
@@ -62,3 +62,9 @@ export function showEventTooltip(nodeId: string): UIThunkAction {
     dispatch(setEventTooltip(nodeId, listenerInfo));
   };
 }
+
+export function viewSourceInDebugger(event: EventInfo): UIThunkAction {
+  return ({ toolbox }) => {
+    toolbox.viewSourceInDebugger(event.url, event.line, event.column, event.sourceId);
+  };
+}
```

Comment:
Looks good. Over time we'll find ways to consolidate some of this logic:

https://github.com/RecordReplay/devtools/blob/master/src/devtools/client/webconsole/actions/toolbox.js#L64

For instance, in the near term we should be able to remove the entire notion of a toolbox object and panel objects: https://github.com/RecordReplay/devtools/issues/1096